src/yuzu/main.cpp

```diff
@@ -2999,6 +3000,12 @@ int main(int argc, char* argv[]) {
     chdir(bin_path.c_str());
 #endif
 
+#ifdef __linux__
+    // Set the DISPLAY variable in order to open web browsers
+    if (QString::fromLocal8Bit(qgetenv("DISPLAY")).isEmpty())
+        qputenv("DISPLAY", ":0");
+#endif
```

Comment:
```suggestion
    if (QString::fromLocal8Bit(qgetenv("DISPLAY")).isEmpty()) {
        qputenv("DISPLAY", ":0");
    }
#endif
```
Style consistency with the project.